### PR TITLE
ci/cd

### PR DESCRIPTION
**Pull request by NavyaNelluri**
ci/cd setup

### CODE REVIEW

**Comment by NavyaNelluri** [comment]
[pullRequest.zip](https://github.com/chintakjoshi/TWA-OSS/files/11928584/pullRequest.zip)

this yaml file will test the files that have uploaded in individual branch and creates merge request to main branch.

**Review by saichetanveeragandham — APPROVED** [review]
LGTM

**Review by chintakjoshi — CHANGES_REQUESTED** [review]
We do not have any dev branch and I think we don't need any CI/CD for merging.

**Comment by NavyaNelluri** [comment]
DEV branch is issue_3 branch

**Comment by harjothkaur** [comment]
LGTM

File: .github/workflows/pullRequest.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# .github/workflows/main.yml
+
+name: Create Merge Request
+
+on:
+  push:
+    branches:
+      - DEV
```
Comment (chintakjoshi) [inline .github/workflows/pullRequest.yml:8]: write your branch name!!!

**Review by saichetanveeragandham — APPROVED** [review]

**Review by chintakjoshi — APPROVED** [review]
lgtm

**Pull request merged**